compat/src/index.js

```diff
@@ -288,16 +288,23 @@ Component.prototype.isReactComponent = {};
  */
 function memo(c, comparer) {
 	function shouldUpdate(nextProps) {
-		return !comparer(this.props, nextProps);
+		let ref = this.props.ref;
+		let updateRef = ref==nextProps.ref;
+		if (!updateRef) {
+			if (typeof ref=='function') ref(null);
```

Comment:
Do you think, basically doing something like:

```js
if (!updateRef) {
  ref.call ? ref(null) : (ref.current = null);
}
```

Would shave some bytes?

Reply:
Ohh that's clever and saves `-2B`! I like your thinking :+1: :tada: